source/locales/ja.json

```diff
@@ -145,7 +145,7 @@
   "file-storage-sync-notes": "<p>クラウドファイルストレージは <strong>Kolab</strong> の <a href='https://git.kolab.org/kolab-chwala/'>Chwala</a>/<a href='https://git.kolab.org/iRony/'>iRony</a> コンポーネントを介して利用することができ、 <a href='https://kolab.org/news/2013/08/07/community-news-meet-chwala-kolabs-new-files-component'>さまざまなストレージバックエンド</a> と統合することができます。 ファイルは、ストレージアクセスオプション、WebDAV、Roundcube を統合した Kolab Web インターフェイスを介してアクセスすることができます。</p><p>Tarsnap が停止した場合にサービスを置き換えることが困難な <a href='https://github.com/nylira/prism-break/issues/555#issuecomment-23216365'>クライアントに対する厳格な著作権</a> のため、PRISM Break で Tarsnap はお勧めしません。</p><p>BitTorrent Sync, MEGA, SpiderOak は、部分的または完全にプロプライエタリのソフトウェアで構築されたサービスです。 彼らがコードベース全体のソースをオープンにするまで、PRISM Break ではお勧めしません。</p><p> クローズドソースソフトウェアでは、個人情報漏洩に関してモラル以外には何もないので、ベンダーを 100% 信頼する必要があります。 あなたが完全性を保証することができたとしても、プロプライエタリなソフトウェアは、ソースコードを調べる目が少ないため、常に不正なセキュリティバグや悪用があります。</p><p>クラウドストレージのもう 1 つの選択肢は、外付けハードドライブと USB フラッシュドライブによるローカルバックアップです。 この方法は、データをネットワークに格納するよりも確実に安全ですが、利便性の面でコストがかかります。</p>",
   "Scheduling": "スケジュール",
   "Online Office Suite": "オンラインオフィススイート",
-  "productivity-notes": "<p><strong>etherpad</strong> プロジェクトが <a href='https://github.com/ether/etherpad-lite/wiki/Sites-that-run-Etherpad-Lite'>etherpad サービスを実行するサイト</a> の一覧を維持しています。 信頼してデータを扱う前に、SSL を使用するサービスだけを選択して、サイトの背景を調査してください。</p><p><a href='#email-accounts'><strong>Riseup</strong></a> もメール、XMPP、チャット、(Up1 経由) データホスティングサービスを提供しています。 すべて <a href='#mesh-networks'>Tor 秘匿サービス</a> アドレスを通じてアクセスすることができます。 これらのアドレスの一覧は <a href='https://www.riseup.net/en/tor#riseups-tor-hidden-services'>ここで利用可能です</a>。</p>",
+  "productivity-notes": "<p><strong>Etherpad</strong> プロジェクトが <a href='https://github.com/ether/Etherpad-lite/wiki/Sites-that-run-Etherpad-Lite'>Etherpad サービスを実行するサイト</a> の一覧を維持しています。 信頼してデータを扱う前に、SSL を使用するサービスだけを選択して、サイトの背景を調査してください。</p><p><a href='#email-accounts'><strong>Riseup</strong></a> もメール、XMPP、チャット、(Up1 経由) データホスティングサービスを提供しています。 すべて <a href='#mesh-networks'>Tor 秘匿サービス</a> アドレスを通じてアクセスすることができます。 これらのアドレスの一覧は <a href='https://www.riseup.net/en/tor#riseups-tor-hidden-services'>ここで利用可能です</a>。</p>",
```

Comment:
This URL change seems to have been done accidentally

Reply:
True! I'll fix that.

(However, GitHub route matching is not case-sensitive, so it doesn't matter much.)